index.html

```diff
@@ -77,6 +77,7 @@ <h4>Subscribe now to get the latest updates!</h4>
         <a target="_blank" href="https://www.linkedin.com/groups/13906372/" class="linkedin" title="Go to Cloud Native Asturias in LinkedIn"><i class="icofont-linkedin"></i></a>
         <a target="_blank" href="https://community.cncf.io/asturias/" class="logo-cncf" title="Go to Cloud Native Asturias events">&nbsp;</a>
         <a target="_blank" href="https://www.facebook.com/cloudnativeasturias/" class="facebook" title="Go to Cloud Native Asturias in Facebook"><i class="icofont-facebook"></i></a>
+        <a target="_blank" href="https://podcasters.spotify.com/pod/show/cloudnativeasturias" class="facebook" title="Go to Cloud Native Asturias Podcast"><i class="icofont-headphone-alt-1"></i></a>
```

Comment:
@Titogelo , better to put the link to Spotify link? https://open.spotify.com/show/0B1a3mw2ZIzPRgd7J44SGk